docs/appendices/release-notes/unreleased.rst

```diff
@@ -157,3 +157,7 @@ Fixes
 
 - Fixed an issue that prevented assigning default expression to ``ARRAY``
   columns.
+
+- Fixed bootstrapping flaws discovered on Kubernetes deployment using the official
```

Comment:
Maybe worth making this more specific.  E.g "Reverted base image change in the Docker image as it broke downstream components which rely on some of the bundled tools like ``rev``"